starter-code/css/main.css

```diff
@@ -6,3 +6,144 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+    margin: 0;
+}
+
+body {
+    font-family: 'Open Sans', sans-serif;
+    font-size: 18px;
+}
+
+h2 {
+    display: inline-block;
+    border-bottom: 2px solid #00B172;
+    padding-bottom: 5px;
+}
+
+header {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    position: fixed;
+    background-color: white;
+    padding: 12px;
+    box-sizing: border-box;
+    top: 0;
+    left: 0;
+}
+
+header img {
+    width: 132px;
+    height: 40px;
+}
+
+header ul {
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+}
+
+header ul li {
+    padding-left: 20px;
+}
+
+section {
+    padding: 20px;
+}
+
+.eyecatcher {
+    height: 600px;
+    background-image: url("../images/landing.jpg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-top: 64px;
+    color: white;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
```

Comment:
if you give your header a height of 10hv and the .eyecatcher a height of 90vh, it will cover the full screen.